compute/backup/manage-backups.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+
+# pip install --upgrade google-api-python-client
+# pip install --upgrade iso8601
+# pip install --upgrade rfc3339
+# pip install --upgrade pytz
+
+import simplejson as json
+import iso8601
+import datetime
+import pytz
+import sys
+from oauth2client.client import GoogleCredentials
+from googleapiclient import discovery
+import re
+import argparse
```

Comment:
Create a short module-level docstring the describes what this does. See [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/storage/api/list_objects.py#L16)